Documentation/AuthConflicsBetweenSDKs.md

```diff
@@ -0,0 +1,47 @@
+# Bridging gap between old and new SDKs
```

Comment:
No difference. Din't notice a new file got added after I fixed the typo in file name. I will remove one